packages/transformers/src/factories/poll.ts

```diff
@@ -0,0 +1,101 @@
+import type { Client, Emoji, PollLayoutType, Poll as LilyPoll, User as LilyUser } from "lilybird";
+import { Message } from "./message.js";
+import { User } from "./user.js";
+
+export class PollMedia {
+    public readonly text: string | undefined;
+    public readonly emoji: Partial<Emoji.Structure> | undefined;
+
+    public constructor(media: LilyPoll.MediaStructure) {
+        this.text = media.text;
+        this.emoji = media.emoji;
+    }
+}
+
+export class PollAnswer {
+    public readonly channelId: string;
+    public readonly messageId: string;
+    public readonly id: number;
+    public readonly media: PollMedia;
+
+    public readonly client: Client;
+
+    public constructor(client: Client, channelId: string, messageId: string, answer: LilyPoll.AnswerStructure) {
+        this.client = client;
+        this.channelId = channelId;
+        this.messageId = messageId;
+
+        this.id = answer.answer_id;
+        this.media = new PollMedia(answer.poll_media);
+    }
+
+    public async fetchVoters(params: {
+        after?: string,
+        /**
+         * 0-100
+         * @default 25
+         */
+        limit?: number
+    } = {}): Promise<Array<User>> {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const voters = await this.client.rest.getPollAnswerVoters(this.channelId, this.messageId, this.id, params);
+
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        return voters.users.map((voter: LilyUser.Structure) => new User(this.client, voter));
+    }
+}
+
+export class PollAnswerCount {
```

Comment:
Do we really need an abstraction for this?

Reply:
We're not in java so yep, we don't. I'll remove it :D